bundles/automation/org.eclipse.smarthome.automation.module.core.test/pom.xml

```diff
@@ -27,6 +27,11 @@
 					<dependencies>
 						<dependency>
 							<type>eclipse-plugin</type>
+							<artifactId>org.eclipse.smarthome.core.scheduler</artifactId>
```

Comment:
this should be removed again as the timer module is not part of module.core anymore